recipes/picojson/all/conanfile.py

```diff
@@ -0,0 +1,29 @@
+from conans import ConanFile, tools
+
+import os
+
+class PicoJSONConan(ConanFile):
+    name = "picojson"
+    license = "BSD 2-Clause \"Simplified\" License"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/kazuho/picojson"
+    description = "A C++ JSON parser/serializer"
+    topics = ("picojson", "json", "header-only", "conan-recipe")
+
+    revision_mode = "scm"
+
+    no_copy_source = True
+
+    _source_subdir_name = "source_subdir"
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        os.rename("picojson-{}".format(self.version), self._source_subdir_name)
+
+    def package(self):
+        self.copy("{}.h".format(self.name), dst="include", src=self._source_subdir_name)
+        self.copy("LICENSE", dst="licenses", src=self._source_subdir_name)
+        print(self.package_folder)
+
+    def package_info(self):
+        self.info.header_only()
```

Comment:
I'm wondering about this one. 

Calling such a helper, might be better future-proof, when Conan might handle more stuff for header_only recipes

Reply:
regarding this one, once I wrote a recipe for a package with CMake modules only, and there was no need for settings too, but I didn't specify header_only() there because it's obviously not applicable in that case. But here specifying header_only() makes the recipe clearer, and as @Croydon mentioned, it can be useful when header_only() implements more actions. Is it a standard approach to not use header_only()? If it is so, could you link the guideline, please?

Reply:
Keep `header_only()` in your recipe https://docs.conan.io/en/latest/howtos/header_only.html#without-unit-tests